Update Nest documentation for Google Home notification setings (#29576)
home-assistant/home-assistant.github.io: source/_integrations/nest.markdown

- *Allow Home Assistant to know when there's a camera event*
- *Allow Home Assistant to know when there's a doorbell event*
- *Other permissions and notification settings in the Nest or Google Home apps*.
</div>

### Google Home App Notification Settings

The Google Home App Notifications settings control not only which notifications are sent to your phone,
but they also affect what gets published to the Pub/Sub feed.

> For example, if you enable “away-only notifications”, HASS will only receive events when your phone is away from home.

Reviewer: ```suggestion
For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.
```

- *Allow Home Assistant to know when there's a camera event*
- *Allow Home Assistant to know when there's a doorbell event*
- *Other permissions and notification settings in the Nest or Google Home apps*.
</div>

### Google Home App Notification Settings

The Google Home App Notifications settings control not only which notifications are sent to your phone,
but also what gets published to the Pub/Sub feed.

For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.

> Another thing that may not be intuitive, is that seeing the event your device history does not mean it's was published to the feed.

Reviewer 2: ```suggestion
Another thing that may not be intuitive, is that seeing the event in your device history does not mean it was published to the feed.
```

> However, if you are getting push notifications the settings are likely working.

Reviewer 2: ```suggestion
However, if you are getting push notifications, the settings are likely working.
```

> Note: The exact settings and affect to feed  may vary by camera model or app version.

Reviewer 2: ```suggestion
Note: The exact settings and effect they have on the feed may vary by camera model or app version.
```

> If you are still not getting notifications you can read this [troubleshooting guide from Google]<!-- textlint-disable -->

Reviewer 2: ```suggestion
If you are still not getting notifications, you can read this [troubleshooting guide from Google]<!-- textlint-disable -->
```

(https://support.google.com/googlenest/answer/9230439#zippy=%2Cyour-camera-detected-something-but-you-didnt-get-a-camera-alert)
<!-- textlint-enable -->

{% details "Google Home App Notification Settings" %}

| Google Home App Setting  | Notes                                                                                 |
| ------------------------ | :-----------------------------------------------------------------------------------: |
| Notifications: Push      | Required for any detection event to be published                                      |

> | Notifications: Away-Only | Events will only be published when user is detected as away from home                 |

Reviewer 2: ```suggestion
| Notifications: Away-Only | Events will only be published when a user is detected as away from home                 |
```

> | Seen: Motion             | Only `motion` events will be published                                                |

Reviewer: Is it "Only" that those events will be published? Seems like this option isn't exclusive with the others

Author: Oops that's not correct.
Revsion here https://github.com/home-assistant/home-assistant.io/pull/29576/commits/f7060e1dab7ebadcd07c4cc066c74a4e0356b8a4

- It is recommended to let Home Assistant create the Pub/Sub subscription for you. However, if you would like more control you can enter a `subscriber_id` in the configuration. See [Subscribe to Events](https://developers.google.com/nest/device-access/subscribe-to-events) for more instructions on how to manually create a subscription and use the full subscription name in the Home Assistant configuration e.g. `projects/gcp-project-name/subscriptions/subscription-id`

> - *Not receiving camera motion and person events*: assuming the integration is correctly configured (e.g. oauth and SDM API is setup correctly, you can see camera streams, and permissions are correctly set in [Partner Connections Manager](https://nestservices.google.com/partnerconnections)), but you are still not seeing events, it's possible you need to adjust Google Home App settings. See the [Cameras: Automation: Google Home App Settings](#google-home-app-settings)

Reviewer 2: ```suggestion
- *Not receiving camera motion and person events*: assuming the integration is correctly configured (for example, the oauth and SDM API are set up correctly, you can see camera streams, and permissions are correctly set in [Partner Connections Manager](https://nestservices.google.com/partnerconnections)): If you are then still not seeing events, it's possible you need to adjust the Google Home App settings. Refer to the [Cameras: Automation: Google Home App Settings](#google-home-app-settings) for details.
```
